src/compiler/checker.ts

```diff
@@ -1794,11 +1794,6 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
         tryGetMemberInModuleExports: (name, symbol) => tryGetMemberInModuleExports(escapeLeadingUnderscores(name), symbol),
         tryGetMemberInModuleExportsAndProperties: (name, symbol) => tryGetMemberInModuleExportsAndProperties(escapeLeadingUnderscores(name), symbol),
         tryFindAmbientModule: moduleName => tryFindAmbientModule(moduleName, /*withAugmentations*/ true),
-        tryFindAmbientModuleWithoutAugmentations: moduleName => {

```

Comment:
In another case of someone doing it wrong, https://github.com/angular/tsickle/blob/2129d363a0562b7ca1acdf55945c22b7d16539a3/src/googmodule.ts#L536

I think it's fine remove this; they'll crash and should just use `tryFindAmbientModule`, but moreso they should file an issue that asks for what they need!